test/controllers/api/v2/repository_sets_controller_test.rb

```diff
@@ -103,6 +103,51 @@ def test_available_repositories
       assert_response :success
     end
 
+    def test_available_repo_sort
+      task = assert_sync_task ::Actions::Katello::RepositorySet::ScanCdn do |product, content_id|
+        product.must_equal @product
+        content_id.must_equal @content_id
+      end
+
+      task.expects(:output).at_least_once.returns(results: [{:substitutions => {:releasever => "5.10", :basearch => "i386"}, :path => ""},
```

Comment:
I see that your mocked data and your expected data are identical - just in a different order. Can you come up with a way to eliminate the duplication and achieve the same result? I think the `shuffle` method will be handy here: https://ruby-doc.org/core-2.2.0/Array.html#method-i-shuffle